2.overview_and_terminology.md

```diff
@@ -281,5 +281,5 @@ This form is not accepted as an alternative for the fully qualified version. It
 
 | Previous Part        | Next Part          |
 | ------------- |-------------|
-|[1. Purpose And Goals](1.purpose_and_goals.md)| [3. The Component Model](4.component.md)|
+|[1. Purpose And Goals](1.purpose_and_goals.md)| [3. Worklaod Definition](3.workload.md)|
```

Comment:
It seems we deleted Component model doc but added back WorkloadDefinition only? Are we going to leave Component for next PR?

Reply:
I don't follow.  The component.md is in this PR.

Reply:
Oh I see, I missed that file.